odpf/optimus/runtime_service.proto

```diff
@@ -179,24 +179,24 @@ service RuntimeService {
     }
     rpc ReplayDryRun(ReplayRequest) returns (ReplayDryRunResponse) {
         option (google.api.http) = {
-            post: "/v1/project/{project_name}/job/{job_name}/replay-dry-run"
+            post: "/v1/project/{project_name}/replay/{job_name}/dryrun"
```

Comment:
sinc dryrun is custom method. it can go as `:dry` 
https://cloud.google.com/apis/design/custom_methods

Reply:
makes sense. let me change it to `:dryrun` and test it

Reply:
@ravisuhag using colon for identifying a custom verb is one way of doing it, other way is to split these custom verbs via slash. The advantage we have here when not using `:verb` is it avoid conflicts with the resource name. For example
```
/v1/project/{project_name}/replay/{job_name}/dryrun
/v1/project/{project_name}/replay/{job_name}:dryrun
```

Also I wonder how will the gateway regex be able to differentiate that when a URL looks like
```
/v1/project/test-project/replay/test-job:dryrun
```
job_name is `test-job:dryrun` or `test-job`? So these colon based verbs are nice when we have a fixed endpoint for example like
```
/v1/project/{project_name}/replay:dryrun
```
and have everything in the POST payload but then we won't be able to do GET calls nicely.

Its very rare we will be going to allow a job name that contains a `/` whereas a job name can contain `:` now or in the future. These colon based verbs can be used if we split them after a slash though like
```
/v1/project/{project_name}/replay/{job_name}/:dryrun
```
But then this looks like more of a choice of convention. For [example](https://docs.github.com/en/rest/reference/gists#star-a-gist) github follows without colon custom verbs. What do you think?

Reply:
Ok cool. fine with slash as well.